tests/test_hf_api.py

```diff
@@ -73,29 +82,35 @@ def test_whoami(self):
 
     def test_presign_invalid_org(self):
         with self.assertRaises(HTTPError):
-            _ = self._api.presign(token=self._token, filename="nested/fake_org.txt", organization="fake")
+            _ = self._api.presign(
```

Comment:
(nit) no need for `_ =` => just `self._api.presign(...)` i good I think

Reply:
👍